### PR TITLE
PLAT-82: Add PostgreSQL support

### PR DESCRIPTION
**Pull request by wikumChamith**
Fixes PostgreSQL compatibility issues in the liquibase.xml

### CODE REVIEW

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -316,17 +316,21 @@
...
         </preConditions>
-        <sql>
-            INSERT INTO
-            scheduler_task_config(name, description, schedulable_class, start_time, start_time_pattern, repeat_interval,
-            start_on_startup, created_by, date_created, uuid)
-            VALUES
-            ('Auto close queue entries Task', 'Auto close queue entries Task', 'org.openmrs.module.queue.tasks.AutoCloseQueueEntryTask',
-            TIMESTAMP(CURDATE()), 'MM/dd/yyyy HH:mm:ss', 86400, TRUE, 1, NOW(), UUID());
-        </sql>
+        <insert tableName="scheduler_task_config">
```
Comment (dkayiwa) [inline api/src/main/resources/liquibase.xml:322]: Did you confirm that that modifying this changeset will not lead to validation checksum errors?

Comment (wikumChamith) [reply api/src/main/resources/liquibase.xml:322]: should I create a separate changeset for Postgresql?

Comment (wikumChamith) [reply api/src/main/resources/liquibase.xml:323]: @dkayiwa updated this by adding a validCheckSum tag.

File: api/src/main/resources/liquibase.xml [hunk 2]
```diff
@@ -400,7 +405,7 @@
         <addForeignKeyConstraint baseColumnNames="visit_id" baseTableName="queue_entry" constraintName="queue_entry_visit_id_fk" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="visit_id" referencedTableName="visit"/>
     </changeSet>
 
-    <changeSet id="migrate_visit_id_to_queue_entry_20231025" author="mseaton">
+    <changeSet id="migrate_visit_id_to_queue_entry_20231025" author="mseaton" dbms="mysql">
```
Comment (ibacher) [inline api/src/main/resources/liquibase.xml:408]:
```suggestion
    <changeSet id="migrate_visit_id_to_queue_entry_20231025" author="mseaton" dbms="mysql,mariadb">
```

Comment (wikumChamith) [reply api/src/main/resources/liquibase.xml]: @ibacher I updated the PR.

**Review by ibacher — APPROVED** [review]

**Pull request merged**